app/views/unattended/provisioning_templates/finish/kickstart_default_finish.erb

```diff
@@ -17,6 +17,7 @@ oses:
   - package_upgrade: boolean (default=true)
   - salt_master: string (default=undef)
   %>
+<%= snippet_if_exists(template_name + " custom pre") %>
```

Comment:
This is still adding empty line, we need to use triming for that not to happen.
```suggestion
<%= snippet_if_exists(template_name + " custom pre") -%>
```

Reply:
Thanks, I didn't realize I had missed this one.